tests/installation/bootloader_hyperv.pm

```diff
@@ -0,0 +1,121 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "installbasetest";
+
+use testapi;
+
+use strict;
+use warnings;
+
+use File::Basename;
+
+use Net::Telnet ();
+
+sub is_jeos() {
+    return get_var('FLAVOR', '') =~ /^JeOS/;
+}
+
+sub run() {
+
+    my $self = shift;
+
+    my $svirt = select_console('svirt');
+    my $name  = $svirt->name;
+
+    # TODO: JeOS defaults to 30 GB on Hyper-V; to be figured out later
+    #my $size_i = get_var('HDDSIZEGB', '30');
```

Comment:
IMHO these TODOs in the code are harmful. Better keep them in a WIP commit in your private development branches or note it down in a ticket if you won't develop and test hyperv+jeos